src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -211,11 +219,47 @@ void onTerminate(@Nullable Module module,
                        @SuppressWarnings("UnusedParameters") @NotNull String... args) {
         ApplicationManager.getApplication().invokeLater(() -> {
           if (workingDir != null && module != null && !module.isDisposed()) {
+            final Project project = module.getProject();
+
             // Enable Dart.
             FlutterSdkUtil.enableDartSupport(module);
-            final FileEditorManager manager = FileEditorManager.getInstance(module.getProject());
-            // Open main.
-            final VirtualFile main = workingDir.findFileByRelativePath("lib/main.dart");
+            final FileEditorManager manager = FileEditorManager.getInstance(project);
+            
+            // Create a basic run configuration.
```

Comment:
Should we check to see if one already exists? Or, asked another way, how would this react if `flutter create` is changed to create an intellij launch configuration as part of its template?